src/components/modals/ButtonsModal.tsx

```diff
@@ -23,14 +23,19 @@ export interface ButtonInfo {
 
 export interface ButtonModalProps<Buttons> {
   bridge: AirshipBridge<keyof Buttons | undefined>
-  title?: string
-  message?: string
-  children?: React.ReactNode
   buttons: Buttons
+  /** Ideally mutually exclusive with message/subtext since children are
+   * rendered in the body of the modal along with message/subtext */
```

Comment:
/** Used to pass non-text children, to be rendered after the title and message but before the buttons themselves. */